hooks/post-command

```diff
@@ -2,7 +2,7 @@
 
 set -uo pipefail
 
-WIZ_DIR="$HOME/.wiz"
+WIZ_DIR="${BUILDKITE_PLUGIN_WIZ_WIZ_DIR:=$HOME/.wiz}"
```

Comment:
I don't see a reason to re-assign the variable (and may cause issues) so using the default parameter substitution instead may be preferable for consistency with the others at least:

```suggestion
WIZ_DIR="${BUILDKITE_PLUGIN_WIZ_WIZ_DIR:-$HOME/.wiz}"
```